tokenclient/tokenFlows.go

```diff
@@ -68,8 +68,7 @@ const (
 // options specifies rest client and its tls config, both can be overwritten.
 // Note: Setup of default tls config is not supported for windows os. Module crypto/x509 supports SystemCertPool with go 1.18 (https://go-review.googlesource.com/c/go/+/353589/)
 func NewTokenFlows(identity *env.Identity, options Options) (*TokenFlows, *ClientError) {
-	t := new(TokenFlows)
-	t.identity = identity
+	t := TokenFlows{Identity: identity}
```

Comment:
@jkbschmid with this i'm unable to return nil, isn't it?
When is what to be preferred?

Reply:
I had to undo the submitted proposal because of compile issues,

Reply:
`new(File)` and `&File{}` are equivalent. From what I've seen `&File{}` is more commonly used.
Perhaps you can also return `TokenFlows` instead of the pointer variant.
But these are just minor things, IMO.

Reply:
I've changed it to
```
t := TokenFlows{
		identity: identity,
		tokenURI: identity.GetURL() + tokenEndpoint,
		options:  options,
	}
```
but left the pointer.
@f-blass what would be your preference?

Reply:
I like your proposal 👍🏼